docs/Tasks.md

```diff
@@ -75,10 +78,13 @@ print(outputs_info)
 #  1.27 MiB     1.21 MiB     important_file2.txt
 
 
+# Download all the generated files.
+output_dir = task.download_outputs()
+
 # Download only those you are interested in:
 output_dir = task.download_outputs(
-    filenames=["important_file1.txt", "important_file2.txt"])
-# 100%|██████████| 4.82M/4.82M [00:00<00:00, 273MiB/s]
+    filenames=["stdout.txt", "stderr.txt"])
+# 100%|██████████| 0.82M/0.82M [00:00<00:00, 53MiB/s]
```

Comment:
these porgression bars will only show when the snippet is executed. They don't belong here.
For future iterations, we should consider having a snippet with the content of the code to be executed followed by the results of executing it. Something like:

```python
# snippet.py
print('Hello world')
```

```bash
$ python snippet.py
Hello World
```

this way we show exactly what the user should expect.